### PR TITLE
Hide sling announcements from lobby

### PR DESCRIPTION
**Pull request by Tayyyyyyy**
Removes the chatting to world stuff for both ascendancy and ascendant transmit, and replaces it with a shadowling ascendant proc.

### CODE REVIEW

**Comment by tigercat2000** [comment]
Is this really necessary? The round is basically over if there's an ascendant.

**Comment by Tayyyyyyy** [comment]
That's true, but I figured it wouldn't hurt to keep lobby a little cleaner and a little more ic out of ooc.

File: code/game/gamemodes/shadowling/shadowling_abilities.dm [hunk 1]
```diff
@@ -867,8 +867,8 @@
 	action_icon_state = "transmit"
 
 /obj/effect/proc_holder/spell/targeted/shadowlingAscendantTransmit/cast(list/targets, mob/user = usr)
-	for(var/mob/living/target in targets)
+	for(var/mob/living/simple_animal/ascendant_shadowling/target in targets)
```
Comment (Fox-McCloud) [inline code/game/gamemodes/shadowling/shadowling_abilities.dm]: You do realize this means you're only sending the message to other ascended shadowlings, right?

Comment (Tayyyyyyy) [inline code/game/gamemodes/shadowling/shadowling_abilities.dm]: My understanding was that this is the target of the spell, which is a self cast. The actual announcement stuff is in a different file and it's borrowed from announce.dm

Comment (Fox-McCloud) [inline code/game/gamemodes/shadowling/shadowling_abilities.dm]: yeah, don't worry about it.

**Comment by SamHPurp** [comment]
In the same way you can't IC in OOC after an ascendant... this change should be made, I disagree Tigercat.

**Comment by tigercat2000** [comment]
@SamHPurp I view it as effectively the same thing as letting people in the lobby know that the escape shuttle has been called- a warning against joining the round, because it'll just waste their time.

**Comment by taukausanake** [comment]
How long is someone going to wait in the lobby though just to know whenever a round is over or not?

**Comment by SamHPurp** [comment]
@tigercat2000 I disagree. When Shadowlings ascend it already calls the shuttle. When you try to join you get the message saying "the station is undergoing evacuation procedures".

While this is certainly not a high priority fix, its inconsistent... and as the PRs been done, why not?

**Comment by Tayyyyyyy** [comment]
Yep, shuttle is called when a shadowling ascends, even if it's not shadowling mode (as I learned during my testing), so the user will know. Even if that weren't the case, if we REALLY wanted to let lobby people know that round is ending, I don't think sling memes is the best way.

**Pull request merged**